include/cantera/oneD/Flow1D.h

```diff
@@ -670,41 +674,166 @@ class Flow1D : public Domain1D
+     * the viscosity evaluated at the midpoints between grid points. In other words,
+     * the value of mu[j] is actually @f$ \mu_{j+1/2} @f$. The transport properties
+     * between grid points are not the average of the adjacent grid points, but rather,
+     * the state of the system at the midpoint is estimated, and then the transport
+     * properties are evaluated at that state (T,P,Y) for example.
+     *
+     * The outer discretization uses a central difference between the j+1/2 and j-1/2
+     * locations.
+     *
+     * @f[
+     *  \frac{dA}{dz} \approx \frac{A_{j+1/2} - A_{j-1/2}}{z_{j+1/2} - z_{j-1/2}}
+     * @f]
+     *
+     * Where the values of @f$ z @f$ are:
+     * @f$ z_{j+1/2} = z_{j} + 0.5*(z_{j+1} - z_j) = 0.5*(z_{j} + z_{j+1}) @f$ and
+     * @f$ z_{j-1/2} = z_j - 0.5*(z_j - z_{j-1}) = 0.5*(z_{j} + z_{j-1}) @f$. The
+     * difference between these two values is
+     * @f$ z_{j+1/2} - z_{j-1/2} = \frac{z_{j+1} - z_{j-1}}{2} @f$.
+     *
+     * Substituting these values into the central difference formula gives:
+     *
+     * @f[
+     *  \frac{d}{dz}\left(\mu \frac{dV}{dz}\right) \approx
+     *   \frac{\mu_{j+1/2} \frac{V_{j+1} - V_j}{z_{j+1} - z_j} -
+     *   \mu_{j-1/2} \frac{V_j - V_{j-1}}{z_j - z_{j-1}}}{\frac{z_{j+1} - z_{j-1}}{2}}
+     * @f]
+     *
+     * @param[in] x The local domain state vector.
+     * @param[in] j The index at which the derivative is computed.
+     */
     double shear(const double* x, size_t j) const {
-        double c1 = m_visc[j-1]*(V(x,j) - V(x,j-1));
-        double c2 = m_visc[j]*(V(x,j+1) - V(x,j));
-        return 2.0*(c2/(z(j+1) - z(j)) - c1/(z(j) - z(j-1)))/(z(j+1) - z(j-1));
+        double A_left = m_visc[j-1]*(V(x, j) - V(x, j-1)) / (z(j) - z(j-1));
+        double A_right = m_visc[j]*(V(x, j+1) - V(x, j)) / (z(j+1) - z(j));
+        return 2.0*(A_right - A_left) / (z(j+1) - z(j-1));
     }
 
-    double divHeatFlux(const double* x, size_t j) const {
-        double c1 = m_tcon[j-1]*(T(x,j) - T(x,j-1));
-        double c2 = m_tcon[j]*(T(x,j+1) - T(x,j));
-        return -2.0*(c2/(z(j+1) - z(j)) - c1/(z(j) - z(j-1)))/(z(j+1) - z(j-1));
+    /**
+     * Compute the conduction term from the energy equation using a central
+     * three-point differencing scheme.
+     *
+     * For the details about the discretization, @see shear().
+     *
+     * @param[in] x The local domain state vector.
+     * @param[in] j The index at which the derivative is computed.
+     */
+    double conduction(const double* x, size_t j) const {
+        double A_left = m_tcon[j-1]*(T(x, j) - T(x, j-1)) / (z(j) - z(j-1));
+        double A_right = m_tcon[j]*(T(x, j+1) - T(x, j)) / (z(j+1) - z(j));
+        return -2.0*(A_right - A_left) / (z(j+1) - z(j-1));
     }
 
+    /**
+     * Array access mapping for a 3D array stored in a 1D vector. Used for
+     * accessing data in the m_multidiff member variable.
```

Comment:
This will link to the corresponding member variable:
```suggestion
     * Array access mapping for a 3D array stored in a 1D vector. Used for
     * accessing data in the #m_multidiff member variable.
```